src/diagnostics.md

```diff
@@ -410,12 +426,12 @@ let mut err = sess.struct_span_err(sp, "oh no! this is an error!");
 if let Ok(snippet) = sess.source_map().span_to_snippet(sp) {
     err.span_suggestion(
         suggestion_sp,
-        "try using a qux here",
+        fluent::example::try_qux_suggestion,
```

Comment:
should first line in this example also be changed to...

    let mut err = sess.struct_span_err(sp, fluent::example::example_error);

Reply:
Fixed!